.pre-commit-config.yaml

```diff
@@ -12,21 +12,20 @@ repos:
       - id: trailing-whitespace
       - id: check-ast
       - id: check-json
+      - id: check-yaml
       - id: check-merge-conflict
-      - id: no-commit-to-branch
```

Comment:
one doesnt have privileges or one knows what he is doing :)

Reply:
Just fyi, you can always skip pre-commit by running `--no-verify` when you commit, so not sure this needs to be removed

Reply:
im aware but i do not like -n